Sources/MessagingPush/RichPush/MessagingPush+RichPush.swift

```diff
@@ -48,43 +48,67 @@ extension MessagingPushImplementation {
         _ request: UNNotificationRequest,
         withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void
     ) -> Bool {
-        logger.info("did recieve notification request. Checking if message was a rich push sent from Customer.io...")
+        logger.info("did receive notification request. Checking if message was a push sent from Customer.io...")
         logger.debug("notification request: \(request.content.userInfo)")
 
-        if sdkConfigStore.config.autoTrackPushEvents,
-           let deliveryID: String = request.content.userInfo["CIO-Delivery-ID"] as? String,
-           let deviceToken: String = request.content.userInfo["CIO-Delivery-Token"] as? String {
+        guard let deliveryID: String = request.content.userInfo["CIO-Delivery-ID"] as? String,
```

Comment:
The code changes done in this file addresses a bug that I found. If you use the new rich composer and send a simple push (just title and body), this code originally would process the simple push, say that the push was not sent from CIO, then would just return without calling the `contentHandler` resulting in the simple push not showing up until ~30 seconds later when the OS would kill our NSE (by calling `serviceExtensionTimeWillExpire`) and then show the push on the device. 

This change I made uses the deliverId and token to determine if the push was sent by CIO - simple or rich. Then, the processing will end early for simple pushes or continue for rich.